moxy/org.eclipse.persistence.moxy/src/org/eclipse/persistence/jaxb/MarshallerProperties.java

```diff
@@ -205,6 +205,43 @@
      */
     public static final String JSON_TYPE_ATTRIBUTE_NAME = JAXBContextProperties.JSON_TYPE_ATTRIBUTE_NAME;
 
+    /**
+     * Disable generation of "item" name in nested arrays in JSON. Settings from binding file have higher priority.
+     *
+     * <p><b>Example</b></p>
+     * <p>Given the following property</p>
+     * <pre>
+     * marshaller.setProperty(MarshallerProperties.JSON_DISABLE_NESTED_ARRAY_NAME, "false");
+     * </pre>
+     * <p>If the property value is set to <b>false</b> (default) the JSON output will be:</p>
+     * <pre>
+     * ...
+     * "char2dArray":[
+     *      {"item":["a","b","c","d"]},
+     *      {"item":["e","f","g","h"]}
+     * ]
+     * ...
+     * </pre>
+     * <p>for following object model</p>
+     * <pre>
+     * ...
+     * &#64;XmlElement(name="char2dArray")
+     * private char[][] char2dArray;
+     * ...
+     * </pre>
+     * <p>If the property value is set to <b>true</b> (default) the JSON output will be:</p>
+     * <pre>
+     * ...
+     * "char2dArray": [
+     *      ["a","b","c","d"],
+     *      ["e","f","g","h"]
+     * ]
+     * ...
+     * </pre>
+     * @since 2.7.5
+     */
+    public static final String JSON_DISABLE_NESTED_ARRAY_NAME = "eclipselink.json.disable-nested-array-name";
```

Comment:
Why not use the JAXB Property instead of a literal?

Reply:
But this property is used only in marshaller scope. JAXB context property means booth unmarshaller and marshaller.

Reply:
Alright, the unmarshaller will properly handle this with or without the property?

Reply:
Yes unmarshaller will handle it automatically without any additional properties.

Reply:
This is then fine by me.